src/ansi-c/cprover_builtin_headers.h

```diff
@@ -22,6 +22,7 @@ void __CPROVER_cleanup(const void *, const void *);
 __CPROVER_bool __CPROVER_get_must(const void *, const char *);
 __CPROVER_bool __CPROVER_get_may(const void *, const char *);
 
+void __CPROVER_printf(const char *format, ...);
```

Comment:
Does the Manual API reference need updating to include the new `__CPROVER_printf`?

Reply:
Fair point, will do!